tool/src/java/org/sakaiproject/gradebookng/tool/panels/AddGradeItemPanelContent.java

```diff
@@ -104,15 +112,39 @@ protected void onUpdate(AjaxRequestTarget target) {
 			}
         };
         released.setOutputMarkupId(true);
-        released.setEnabled(!assignment.getObject().isCounted());
+        released.setEnabled(!assignmentModel.getObject().isCounted());
         add(released);
         
         //if checked, release must also be checked and then disabled
-        final AjaxCheckBox counted = new AjaxCheckBox("counted", new PropertyModel<Boolean>(assignment, "counted")) {
+        final AjaxCheckBox counted = new AjaxCheckBox("counted", new PropertyModel<Boolean>(assignmentModel, "counted")) {
 			private static final long serialVersionUID = 1L;
 
 			@Override
 			protected void onUpdate(AjaxRequestTarget target) {
+				refreshState(target);
+			}
+
+			@Override
+			public void onEvent(IEvent<?> event) {
```

Comment:
I'm curious as to why this can't be done in the existing AjaxFormComponentUpdatingBehavior of the category dropdown? There is already code there to handle change events and to update certain fields.